libs/application/templates/parental-leave/src/fields/Review/index.tsx

```diff
@@ -0,0 +1,403 @@
+                    name={'employer.contactId'}
+                    label={'Contact social security nr'}
+                    ref={register}
+                  />
+                </GridColumn>
+              </GridRow>
+            </Box>
+          </AccordionItem>
+
+          <AccordionItem id="id_4" label="Your leave rights">
+            <Box paddingY={4}>
+              <GridRow>
+                <GridColumn span="12/12">
+                  <YourRightsBoxChart application={application} />
+                </GridColumn>
+              </GridRow>
+            </Box>
+          </AccordionItem>
+
+          <AccordionItem id="id_4" label="Your periods">
+            <Box paddingY={4}>
+              <GridRow>
+                <GridColumn span="12/12">
+                  <Timeline
+                    initDate={dobDate}
+                    title="Expected birth date"
+                    titleSmall="Birth date"
+                    periods={formatPeriods(
+                      application.answers.periods as Period[],
+                      otherParentPeriods,
+                    )}
+                    onDeletePeriod={() => {
+                      // TODO
+                    }}
+                  />
+                </GridColumn>
+              </GridRow>
+            </Box>
+          </AccordionItem>
+
+          <AccordionItem id="id_4" label="Payment plan">
+            <Box paddingY={4}>
+              <GridRow>
+                <GridColumn span="12/12">
+                  <PaymentsTable application={application} />
+                </GridColumn>
+              </GridRow>
+            </Box>
+          </AccordionItem>
+
+          <AccordionItem id="id_4" label="Share information">
+            <Box paddingY={4}>
+              <GridRow>
+                <GridColumn span="12/12">
+                  <Box marginTop={1} marginBottom={2} marginLeft={4}>
+                    <Text variant="h5">
+                      Do you want to share your leave information with the other
+                      parent?
+                    </Text>
+                  </Box>
+
+                  <RadioController
+                    emphasize={true}
+                    id={'shareInformationWithOtherParent'}
+                    disabled={false}
+                    name={'shareInformationWithOtherParent'}
+                    defaultValue={
+                      getValueViaPath(
+                        application.answers,
+                        'shareInformationWithOtherParent',
+                      ) as string[]
+                    }
+                    options={[
+                      { label: 'Yes', value: 'yes' },
+                      { label: 'No', value: 'no' },
+                    ]}
+                  />
+                </GridColumn>
+              </GridRow>
+            </Box>
+          </AccordionItem>
+        </Accordion>
+      </Box>
+    </div>
+  )
+}
+
+export default Review
```

Comment:
I like the changes here, the only thing I would add later is error handling. I think we need that at some point. 

More importantly, I think the application system should know how to dynamically render a summary screen so this repetitive coding is unnecessary, but that can wait until later :)

Reply:
Agree, both error handling and validation. 
And yeah I think it would be nice to make it part of the general Application System.